easybuild/easyconfigs/m/MRIcron/MRIcron-06_22_15.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "PackedBinary"
+
+name = 'MRIcron'
+version = '06_22_15'
+
+homepage = 'http://www.mccauslandcenter.sc.edu/mricro/mricron/'
+description = """ MRIcron allows viewing of medical images. It 
+includes tools to complement SPM and FSL. Native format is NIFTI 
+but includes a conversion program (see dcm2nii) for converting DICOM images. 
+Features layers, ROIs, and volume rendering.  """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Automatic Download doesn't work for this software as you need to accept their license, 
+# so you can download it manually at https://www.nitrc.org/frs/download.php/7765/ 
+# and place it in a folder where Easybuild can see it.
+sources = ['lx.zip']
```

Comment:
Thanks for including the checksum.

On 2nd consideration, I would take it one step further, i.e. force people to rename the downloaded `lx.zip` to something like `lx-20150601.zip`, since there are frequent releases...

@rtripath89 thoughts?